docs/user-guide.rst

```diff
@@ -350,6 +350,19 @@ Both of those outputs can then be used in the tarmak configuration:
       additionalIAMPolicies:
       - ${elasticsearch_shipping_policy_arn}
 
+EBS Encryption
+~~~~~~~~~~~~~~
+
+AWS offers encrypted EBS (Elastic Block Storage). Details regarding EBS can be
+found `here<https://aws.amazon.com/ebs/details/>`_, while Amazon EBS Encryption is described `here<https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html>`_. 
+The following `tarmak.yaml` snippet shows how to enable encrypted EBS.
```

Comment:
Maybe mention that EBS encryption is off by default and that we use the default KMS customer master keys